include/dump_offload.hpp

```diff
@@ -351,8 +374,8 @@ inline void requestRoutes(App& app)
 
```

Comment:
When you hit "Unable to extract the dump id" at line 350, please return not_found error

Reply:
done